cmd/syncIndexer.go

```diff
@@ -117,6 +117,7 @@ func (i *folderIndexer) store(storedObject StoredObject) (err error) {
 			err := fmt.Errorf("FileName [%s] under Folder [%s] already present in map", lcFileName, lcFolderName)
 			return err
 		}
+		size = storedObjectSize(storedObject)
```

Comment:
actually I just saw, we should update size at L109 and L162 since the objectIndexer will also consume memory and we have one for each directory which will become significant for small dir workload.